src/targz_sizes.c

```diff
@@ -120,8 +126,24 @@ int main(int argc, char** argv) {
         }
     }
 
+    unsigned int max_filename_blocks;
+    if ( LEVEL_IS_TEST( verbosity ) ) {
+        // when targz_sizes is in *test* mode, set max_filename_blocks
+        // very small (2).  This makes it easier to test the case where
+        // a long filename would overrun the buffer
+        max_filename_blocks=2;
```

Comment:
Again, sorry about the formatting inconsistencies.  Addressed in commit: 311a6eebb9e5cf57e4e9a6d74d70e0f733f905ce